pkg/cmd/admin/router/router.go

```diff
@@ -470,6 +479,9 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 			ContainerPort: cfg.StatsPort,
 			Protocol:      kapi.ProtocolTCP,
 		}
+		if cfg.HostPorts {
```

Comment:
doesn't this need to check `!cfg.HostNetwork`? the flag help seemed to indicate it only does something if HostNetwork is false